otx/cli/builder/builder.py

```diff
@@ -155,12 +156,11 @@ def build_task_config(
         """
 
         # Create OTX-workspace
-        if workspace_path is None:
-            workspace_path = f"./otx-workspace-{task_type}"
-            if model_type:
-                workspace_path += f"-{model_type}"
-        workspace_path = workspace_path if isinstance(workspace_path, Path) else Path(workspace_path)
-        workspace_path.mkdir(exist_ok=False)
+        # Check whether the workspace is existed or not
+        # FIXME: better variable name? is_autoconfig_enabled?
+        workspace_path = Path(workspace_path)
+        if not exist:
+            workspace_path.mkdir(exist_ok=False)
```

Comment:
It looks like this could be replaced with `workspace_path.mkdir(exist_ok=exist)` without needing an if statement.

Reply:
How do you like this comment?

Reply:
Since we changed the workflow of `builder.py`, `exist` argument is not needed anymore.
![image](https://user-images.githubusercontent.com/80735344/214537048-291fffea-f5b8-4e77-882a-21828aff2800.png)